e2e/vmss.go

```diff
@@ -110,7 +110,10 @@ func addPodIPConfigsForAzureCNI(vmss *armcompute.VirtualMachineScaleSet, vmssNam
 		}
 		podIPConfigs = append(podIPConfigs, ipConfig)
 	}
-	vmssNICConfig := vmss.Properties.VirtualMachineProfile.NetworkProfile.NetworkInterfaceConfigurations[0]
+	vmssNICConfig, err := getVMSSNICConfig(vmss)
+	if err != nil {
+		return fmt.Errorf("unable to get vmss nic: %s", err)
```

Comment:
should you be wrapping the error with %w instead of %s?

Reply:
we should actually go back and change this everywhere